Source/Blazorise/EventHandlers.cs

```diff
@@ -15,4 +17,10 @@ namespace Blazorise
     public delegate void ClearAllValidationsEventHandler();
 
     public delegate void ValidationsStatusChangedEventHandler( ValidationsStatusChangedEventArgs e );
+
+    [EventHandler( "onmouseenter", typeof( MouseEventArgs ), true, true )]
+    [EventHandler( "onmouseleave", typeof( MouseEventArgs ), true, true )]
+    public static class EventHandlers
```

Comment:
What is the usage of this class?

Reply:
I think its mostly for Intellisense.. But I needed support for `onmouseenter` and `onmouseleave`.
Found the solution here: https://github.com/dotnet/aspnetcore/issues/13104

Reply:
First time I hear about this solution haha. In that case just write a comment describing what is used for.